podcasts/DefaultPlayer.swift

```diff
@@ -396,7 +396,8 @@ class DefaultPlayer: PlaybackProtocol, Hashable {
         let peakLimiterRenderCallback: AURenderCallback = { inRefCon, _, _, _, inNumberFrames, ioData -> OSStatus in
             if ioData == nil { return -1 }
 
-            let referenceToSelf = unsafeBitCast(inRefCon, to: DefaultPlayer.self)
+            let reference = Unmanaged<DefaultPlayer>.fromOpaque(inRefCon)
+            let referenceToSelf = reference.takeUnretainedValue()
```

Comment:
Since the crash is a `EXC_BAD_ACCESS`, would it make any sense to try `takeRetainedValue` here and `passRetained` in the other calls?

In [this thread](https://forums.swift.org/t/using-unmanaged-and-unsafebitcast-to-interface-with-c/30864), Quinn ways:

> If you can guarantee that the instance of Button outlasts any possibility that widget might call your handler, it’s OK to use an unretained reference. If not, it’s best to use a retained reference (although you’ll then need a way to release that reference when you’re done).

I just wonder if the crash indicates that the player may be going away and taking a retained version could help. Maybe it's best to save that for a future PR if this crash isn't resolved?

Reply:
> I also realized that on other methods of this class we are already following this approach

Ah, this is good to know! I agree with the retain cycle concern, although I would think that would be easy for us to check. If these other spots aren't showing up in crash logs it's probably best to just copy them for now.